### ISSUE TITLE
Disable on Full Screen windows (but keep maximized windows enabled)

### ISSUE THREAD

**Issue by hrqmonteiro** [matinlotfali/KDE-Rounded-Corners#292]
Is there any way to disable outline and rounded on full screen windows (not maximized), like when you put a video on Firefox or VLC on fullscreen:

![image](https://github.com/user-attachments/assets/7f501c55-5503-4afe-b433-3140df7324ac)

Cause like this it looks very strange. I want the rounded on maximized windows, like:

![image](https://github.com/user-attachments/assets/d6be7ed4-ccaa-47ef-966f-13ed6b3819ab)

But i don't want it on fullscreen cause it just looks very distracting

**Comment by matinlotfali** [comment]
Do you have this checkbox checked in the config window?
![image](https://github.com/user-attachments/assets/a7646135-9c6c-4bbe-8ecb-43c277fa5c20)

**Comment by hrqmonteiro** [comment]
> Do you have this checkbox checked in the config window? ![image](https://private-user-images.githubusercontent.com/7337168/380443107-369e0a4e-65f4-4d65-b9bc-ee9dfdab6768.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk5NjcyNjIsIm5iZiI6MTcyOTk2Njk2MiwicGF0aCI6Ii83MzM3MTY4LzM4MDQ0MzEwNy0zNjllMGE0ZS02NWY0LTRkNjUtYjliYy1lZTlkZmRhYjY3NjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjZUMTgyMjQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWNlNzMwNTg2Nzk0NmQ0MmQ0MmI4M2YwMmUwMDgzYTczYmQyOTIxZjUyOThlNTU2MWYxZDVhZGE1Mjk5Y2Y5OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.4AiVOdUG0p5kgJtsTjEvquzqjJpRy6sISFFIB0lQ_IA)

No, because i am asking if i can have them on Maximized but not on Full Screen, as i said in my text

Maximized is different than Full Screen

**Comment by matinlotfali** [comment]
Oh, I see now. Sorry for not reading your text more carefully. 

At this moment, I am treating Maximized and Fullscreen windows the same. I can check changing the code to see if I can make the distinction without affecting the OSD windows (windows that draw borders themselves)

**Comment by tockudex** [comment]
As much as I'm also interested in the distinction of maximized and fullscreen windows (I was about to open a similar issue), I wondered if you could provide some information/details on how you riced your KDE like that @hrqmonteiro.

(Sorry, I know I'm not supposed to drift off-topic, but I did not know where to ask otherwise.)

**Comment by hrqmonteiro** [comment]
> As much as I'm also interested in the distinction of maximized and fullscreen windows (I was about to open a similar issue), I wondered if you could provide some information/details on how you riced your KDE like that @hrqmonteiro.
> 
> (Sorry, I know I'm not supposed to drift off-topic, but I did not know where to ask otherwise.)

Sure!

The colors is [Everforest](https://github.com/Serge2702/KDE-Everforest), to configure the panel like that i am using [Panel Colorizer](https://github.com/luisbocanegra/plasma-panel-colorizer), and the tilling functionality comes from [Khronkite](https://github.com/esjeon/krohnkite)

**Comment by shdbwa** [comment]
Is there any kwin rule to get the most of it, without disabling the compositor ?

**Comment by matinlotfali** [comment]
@tockudex I will submit a PR soon, and I need everyone here to test it.
